frontend/src/components/processMeasurements/ProcessMeasurementsDetailContent.js

```diff
@@ -31,7 +33,10 @@ const ProcessMeasurementsDetailContent = ({
   samplesByID,
   usersByID,
```

Comment:
usersById is not used

Reply:
I wrote that? 🤔

Reply:
Ah, no I guess not. But, it's nice to follow the Boy Scout Principal - leave the code cleaner than when you found it (if the cleanup is easy).